src/i18n.rs

```diff
@@ -149,55 +149,64 @@ impl TeamHelperParam {
 impl HelperDef for TeamHelper {
     fn call<'reg: 'rc, 'rc>(
         &self,
-        h: &Helper<'reg, 'rc>,
+        h: &Helper<'rc>,
         _: &'reg Handlebars,
         context: &'rc Context,
         rcx: &mut RenderContext<'reg, 'rc>,
         out: &mut dyn Output,
     ) -> HelperResult {
         let Some(name) = h.param(0) else {
-            return Err(RenderError::new(
+            return Err(RenderErrorReason::ParamNotFoundForIndex(
                 "{{team-text}} must have at least two parameters",
-            ));
+                0,
+            )
```

Comment:
This wasn't a hard error, only a deprecation warning. I hope the variants I chose make some sense. It's a bit late.